packages/app/src/components/FormFieldItems.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Avatar,
+  HookedInputCheckboxGroup,
+  ListItem,
+  Text,
+  type HookedInputCheckboxGroupProps
+} from '@commercelayer/app-elements'
+
+import type { LineItem } from '@commercelayer/sdk'
+import { useMemo } from 'react'
+interface Props {
+  lineItems: LineItem[]
+}
+
+export function FormFieldItems({ lineItems }: Props): JSX.Element {
+  const options: HookedInputCheckboxGroupProps['options'] = useMemo(
+    () =>
+      lineItems.map((item) => ({
+        value: item.id,
+        content: (
+          <ListItem
+            alignIcon='center'
+            alignItems='center'
+            borderStyle='none'
+            icon={
+              item.image_url != null ? (
+                <Avatar
+                  alt={item.name ?? ''}
+                  size='small'
+                  src={item.image_url as `https://${string}`}
```

Comment:
it would be nice to have this type on app-elements, what do you think?

Reply:
Good idea.. :) @marcomontalbano @gciotola how we can handle this? is there a right place to store this kind of type? Directly inside the `Avatar` or anywhere else?

Reply:
Maybe by doing `item.image_url as AvatarProps['src']` ?
It would be great if it could be automatically typed from the `item` resource, but I don't think it's possible at the moment.

also a type-guard helper could be handy to be exported from elements.
```ts
function isValidAvatarSrc (url?: string): url is AvatarProps['src'] => boolean { 
  ... 
}
```